deployments/deploy-ssh.sh

```diff
@@ -271,7 +276,8 @@ spec:
             "--provisioner", "$PROVISIONER",
             "--config-map", "$CONFIGMAP",
             "$UPGRADE_ENDPOINT_URL_ARG",
-            "$UPGRADE_ENDPOINT_VERSION_ARG" ]
+            "$UPGRADE_ENDPOINT_VERSION_ARG",
+            "$APP_DESTROY_TIMEOUT_ARG" ]
```

Comment:
don't use empty string as an argument here, use --destroy-app-timeout=$APP_DESTROY_TIMEOUT

Reply:
IMHO we already have a default set in torpedo side, there is no purpose to pass `--destroy-app-timeout=$APP_DESTROY_TIMEOUT` when `APP_DESTROY_TIMEOUT` is not provided